app/src/main/java/com/example/joshuarodriguez/mvpcalculator/mvp/mvp/view/CalculatorView.java

```diff
@@ -56,136 +56,134 @@
     @BindView(R.id.button_equals) Button mButtonEquals;
     @BindView(R.id.button_clear) Button mButtonClear;
 
-    public CalculatorView(AppCompatActivity activity, Bus eventBus) {
+    public CalculatorView(AppCompatActivity activity) {
         super(activity);
-        this.eventBus = eventBus;
         ButterKnife.bind(this, activity);
-        hideKeyboard();
     }
 
     @OnClick(R.id.button_number_zero)
     public void numericButtonZeroPressed() {
-        eventBus.post(new CalculatorNumericButtonPressedEvent(
-                ViewUtil.getStringResourceValue(getActivity(), R.string.calculator_button_value_zero)
+        RxBus.post(new CalculatorNumericButtonPressedObserver.NumericButtonValue(
+                getStringResource(R.string.calculator_button_value_zero)
         ));
```

Comment:
You should not pass this string value. Because is responsability of presenter. Instead, post an new CalculatorNumericButtonPressedObserver.NumericButtonValue(), and the presenter when receive that, already know the string value to use. Review the other cases.